main.go

```diff
@@ -17,21 +17,23 @@ const (
 )
 
 var (
-	addr = flag.String("web.listen-address", ":9445", "Address to listen on for web interface and telemetry.")
+	addr            = flag.String("web.listen-address", ":9445", "Address to listen on for web interface and telemetry.")
 	disableFanSpeed = flag.Bool("disable-fanspeed", false, "Disable fanspeed metric")
 
 	labels = []string{"minor_number", "uuid", "name"}
 )
 
 type Collector struct {
 	sync.Mutex
-	numDevices  prometheus.Gauge
-	usedMemory  *prometheus.GaugeVec
-	totalMemory *prometheus.GaugeVec
-	dutyCycle   *prometheus.GaugeVec
-	powerUsage  *prometheus.GaugeVec
-	temperature *prometheus.GaugeVec
-	fanSpeed    *prometheus.GaugeVec
+	numDevices         prometheus.Gauge
+	usedMemory         *prometheus.GaugeVec
+	totalMemory        *prometheus.GaugeVec
+	dutyCycle          *prometheus.GaugeVec
+	powerUsage         *prometheus.GaugeVec
+	temperature        *prometheus.GaugeVec
+	fanSpeed           *prometheus.GaugeVec
+	nvlinkThroughputTx *prometheus.GaugeVec
```

Comment:
What does Tx suffix stand for ?

Reply:
Tx and Rx stand for transmitted and received respectively. They're pretty widely used labels across NVML. The Grafana panel actually spell this out.